src/test/jetpack-editor-setup.js

```diff
@@ -47,21 +47,6 @@ const jetpackEmbedVariations = [
 	'smartframe',
 ];
 
-// Jetpack blocks are registered when importing the editor extension module.
-// Since we need to register the blocks multiple times for testing,
-// it's required to isolate modules for re-importing the editor extension multiple times.
-const registerJetpackBlocksIsolated = ( props ) => {
-	jest.isolateModules( () => {
-		registerJetpackBlocks( props );
-	} );
-};
-// Similarly to Jetpack blocks, Jetpack embed variations also require to isolate modules.
-const registerJetpackEmbedVariationsIsolated = ( props ) => {
-	jest.isolateModules( () => {
-		registerJetpackEmbedVariations( props );
-	} );
-};
```

Comment:
For the same reason as outlined in [this comment](https://github.com/wordpress-mobile/gutenberg-mobile/pull/4499/files#r789688147), I removed the use of `isolateModules`. In this case, the main purpose of using this function was to assure that Jetpack/Layout-grid blocks are registered on each test, as they are registered upon module importation. However, I realized after removing that function that it's no longer required to isolate the modules, as blocks are registered on every test.